esp/src/src-react/index.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import { initSession } from "src/Session";
+
+import "css!dojo-themes/flat/flat.css";
+import "css!hpcc/css/ecl.css";
+import "css!hpcc/css/hpcc.css";
+
+declare const dojoConfig: any;
+
+const baseHost = "";
+const hashNodes = location.hash.split("#");
+
+dojoConfig.urlInfo = {
+    baseHost,
+    pathname: location.pathname,
+    hash: hashNodes.length >= 2 ? hashNodes[1] : "",
```

Comment:
Are we free to use `??` within `/src-react/*` files?
This line could be: `hashNodes[1] ?? ""`

Reply:
If this was master we can - but 7.x we can't.